saferpay_test/src/Controller/SaferpayTestController.php

```diff
@@ -29,29 +32,46 @@ public function __construct() {
   /***
    * With CreatePayInit() a payment link can be generated.
    *
-   * @return RedirectResponse
+   * @param Request
```

Comment:
param and return should use the full namespace for the classes and include a short description on the next line (indented by two spaces)